.github/workflows/sync_docs.yml

```diff
@@ -24,11 +24,14 @@ jobs:
           curl --proto '=https' --tlsv1.2 -fsSL https://static.pantsbuild.org/setup/get-pants.sh | bash
           echo "$HOME/.local/bin" >> $GITHUB_PATH
       - name: Generate help JSON
-        # Run the command twice, since the first run might put extraneous crap on stdout
+        # First run `help-all` which will contain the list of every backend in the `name_to_backend_help_info` object,
+        # then run it again with every backend enabled.
+        # NB: We run the command initially twice, since the first run might put extraneous crap on stdout.
```

Comment:
Btw, is this something we can file a Pants bug about, since it's seems... unhelpful?

Reply:
I think it might be a scie-pants issue but haven't had time to dig in

Reply:
I think this covers it: https://github.com/pantsbuild/pants/issues/20315